examples/demo_vni.pp

```diff
@@ -0,0 +1,23 @@
+# Manifest to demo cisco_vni provider
```

Comment:
- Can you combine all vxlan related providers into one file called
   demo_vxlan.pp? Currently this would includecisco_vxlan_global and
   cisco_vni. We will add cisco_vxlan_vtep and cisco_vxlan_vtep_vni later.
   - Add an entry in demo_all.pp to call demo_vxlan.pp

Done.

On Wed, Dec 16, 2015 at 4:27 AM, Mike Wiebe <notifications@github.com>
wrote:

> In examples/demo_vni.pp
> <https://github.com/cisco/cisco-network-puppet-module/pull/110#discussion_r47769561>
> :
>
> > @@ -0,0 +1,23 @@
> > +# Manifest to demo cisco_vni provider
>
> Couple things here.
>
>    - Can you combine all vxlan related providers into one file called
>    demo_vxlan.pp? Currently this would include cisco_vxlan_global and
>    cisco_vni. We will add cisco_vxlan_vtep and cisco_vxlan_vtep_vni later.
>    - Add an entry in demo_all.pp to call demo_vxlan.pp
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cisco/cisco-network-puppet-module/pull/110/files#r47769561>
> .
>